src/pages-and-resources/discussions/app-list/AppCard.jsx

```diff
@@ -51,12 +52,12 @@ function AppCard({
         </div>
         <Card.Subtitle className="mb-3 text-muted">{supportText}</Card.Subtitle>
         <Card.Text>{intl.formatMessage(messages[`appDescription-${app.id}`])}</Card.Text>
-        <ExtraSmall>
+        <Responsive maxWidth={575.99}>
```

Comment:
I think these numbers are not self-explanatory, we need to wrap them inside some variable or import the breakpoints from paragon for use.